Ukoly/DU_7/Hangman/Hangman.py

```diff
@@ -0,0 +1,101 @@
+from random import choice
+from Extras import get_pictures
+
+list_of_words = ['praha', 'olomouc']
+
+
+def format_of_field_single(field, position, sign):
+    start = field[:position * 2]
+    end = field[(position + 1) * 2:]
+    field = '{}{}{}'.format(start, sign, end)
+    return field
+
+
+def format_of_field_many(field, position, sign):
+    while field.count(sign) != len(position):
+        for i in position:
+            start = field[:i * 2]
+            end = field[(i + 1) * 2:]
+            field = '{}{}{}'.format(start, sign, end)
+    return field
+
+
+def guess():
+    letter = input('Guess the letter: ')
+    return letter
+
+
+def hra():
+    word = choice(list_of_words)
+    field = len(word) * '_ '
+    counter = 0
+    image(field, counter)
+    while '_' in field:
+        letter = guess()
+        if len(letter) != 1 and letter.isalpha():
+            print('Please input only one letter')
+        if letter in word and letter not in field:
+            if word.count(letter) == 1:
+                field = move_positive_single(word, letter, field)
+            else:
+                field = move_positive_many(word, letter, field)
+        else:
+            counter += 1
+            if counter == 9:
+                image(field, counter)
+                result = 'defeat'
+                evaluation(result)
+                return result
+        image(field, counter)
+    result = 'win'
+    evaluation(result)
+    return result
+
+
+def evaluation(result):
+    if result == 'win':
+        print('Well done, you won!')
+    else:
```

Comment:
Tento `else` je zbytočné zanorenie/odsadenie, nakoľko si mohla v `if` podmienke použiť `return`...

Reply:
Tady asi úplně nevím, jak přesně to myslíš, resp. jak bych to return měla použít.